cosmic-update.jenkinsfile

```diff
@@ -131,15 +116,13 @@ pipeline
 			{
```

Comment:
I can't actually comment above (the email stage) because technically it wasn't changed in these most recent changes, but the `${currentRelease}` variables need to be done so that `releaseVersion` is retrieved via `utils.getReleaseVersion()` and the `currentRelease` variables updated accordingly

Reply:
If you view the whole file, you should be able to comment on whatever line you like. And since _everything_ is new in this PR, you should be able to comment anywhere.

Reply:
Yea I think it was just because I was looking at it through the 'changes since you last reviewed' view